web/pgadmin/browser/server_groups/servers/databases/schemas/collations/tests/pg/16_plus/alter_collation.sql

```diff
@@ -3,10 +3,10 @@
 -- DROP COLLATION IF EXISTS testschema."Cl1_$%{}[]()&*^!@""'`\/#a";
 
 CREATE COLLATION IF NOT EXISTS testschema."Cl1_$%{}[]()&*^!@""'`\/#a"
-    (LC_COLLATE = 'C', LC_CTYPE = 'C');
+    (LC_COLLATE = 'C', LC_CTYPE = 'C', PROVIDER = 'c', DETERMINISTIC = true);
 
 ALTER COLLATION testschema."Cl1_$%{}[]()&*^!@""'`\/#a"
-    OWNER TO <OWNER>;
+    OWNER TO postgres;
```

Comment:
why is this changed? I think it is required for epas servers. Did you run tests on epas?